### PR TITLE
fix: AppService in the UI by adding a second auth field

### LINKED ISSUE #672: When accessing the AppService menu, an error occurs: "Oops!!! registry center service is unavailable!"

### PR DESCRIPTION
**Pull request by dovholuknf**
closes #672 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)

## Testing Instructions
* check out this branch
* issue `make docker` to create a new ui image
* enable security
* update `docker-compose.yml` with the local dev UI image
* bring up the docker environment
* open the UI see the new dialog
* from `edgex-compose` issue `make get-token` use this as the "Gateway Token" and `make get-consul-acl-token` for the "ACL Token"

## New Dependency Instructions (If applicable)
none

### CODE REVIEW

**Comment by dovholuknf** [comment]
Thanks to @rgallettonf for helping out!

**Comment by lenny-intel** [comment]
@dovholuknf , please fix you commit message and PR title to add `:` after `fix`

**Comment by dovholuknf** [comment]
> @dovholuknf , please fix you commit message and PR title to add `:` after `fix`

is there some kind of strange bug by having the word 'fix' twice? i commited with `fix: fix AppService`.... 
image

File: web/src/app/guards/auth/login/login.component.html [hunk 1]
```diff
@@ -25,22 +25,32 @@
...
                         </div>
                     </div>
+
+                  <label for="registryToken" class="col-form-label">
+                    <span class="mr-1" i18n>ACL Token</span>
+                  </label>
+                  <div class="">
+                    <input type="password" class="form-control" id="registryToken" name="registryToken"  [class.is-invalid]="!registryTokenIsValid" [(ngModel)]="registryToken" (ngModelChange)="onInput()">
+                    <div class="invalid-feedback">
+                      <small i18n>the registry token is invalid!</small>
```
Comment (lenny-intel) [inline web/src/app/guards/auth/login/login.component.html:43]:
```suggestion
                      <small i18n>the ACL token is invalid!</small>
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]
One minor issue. Thanks for the PR!

File: web/src/app/guards/auth/login/login.component.html (hunk 1 above)
Comment (dovholuknf) [reply web/src/app/guards/auth/login/login.component.html]: changed

**Comment by lenny-intel** [comment]
@dovholuknf , I think we need to make the ACL token optional. Something like

`ACL Token (required for App Service)`

**Comment by lenny-intel** [comment]
We also need to update the doc at [How to get access token?](https://docs.edgexfoundry.org/3.0/security/Ch-Authenticating/#how-to-make-authenticated-edgex-calls) to include how to get the ACL token.

**Comment by dovholuknf** [comment]
two new commits pushed. here's a gif of it in action:

new-edgex-ui-login

**Review by lenny-intel — APPROVED** [review]
LGTM and works great.

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**